platforms/ios/ios.json

```diff
@@ -35,6 +35,10 @@
                         {
```

Comment:
@cloworm why are changes to `platforms` and `plugins` still getting tracked in your commits? I thought they were in `gitignore`?